<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ApplyFast</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabLayout">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabPersonalDetails">
       <attribute name="title">
        <string>Personal Details</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>11</y>
          <width>981</width>
          <height>190</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Given Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Middle Name</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Address Line 2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QCheckBox" name="chkPreferWest">
           <property name="text">
            <string>Western name order for preferred name</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Postcode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>State/Province</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Address Line 1</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="txtAddress2"/>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="txtPreferredName"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Surname</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="txtPostcode">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>100</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtState"/>
           </item>
          </layout>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Preferred Name</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="txtAddress1"/>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="txtMiddleName">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="chkEastAsian">
           <property name="text">
            <string>Use East Asian order</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="txtCity"/>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="chkNoMidName">
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="chkIsMononymous">
           <property name="text">
            <string>I have only one name</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="txtSurname"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Country</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="txtGivenName"/>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>City</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLineEdit" name="txtCountry"/>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Email Address</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="txtEmail"/>
         </item>
         <item row="6" column="2" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Phone Number</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Website</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>LinkedIn Profile</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="7" column="2" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="txtPhone">
             <property name="maximumSize">
              <size>
               <width>115</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtWebsite"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtLinkedIn"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>390</y>
          <width>979</width>
          <height>221</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLabel" name="label_17">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Achievements</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLineEdit" name="txtSkill"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnAddSkill">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_16">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Skills</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLineEdit" name="txtAchievement"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnAddAchievement">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="btnSave">
           <property name="text">
            <string>Save Personal Details</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QListView" name="skillsListView"/>
         </item>
         <item row="2" column="1">
          <widget class="QListView" name="achievementsListView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>207</y>
          <width>979</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Personal Profile</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textPersonalProfile">
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="acceptRichText">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabWorkExp">
       <attribute name="title">
        <string>Work Experience</string>
       </attribute>
       <widget class="QListView" name="workExpListView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>256</width>
          <height>591</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>0</y>
          <width>411</width>
          <height>621</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="5" column="1" rowspan="2" colspan="3">
          <widget class="QComboBox" name="cmbEndMonth">
           <item>
            <property name="text">
             <string>January</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>February</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>March</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>April</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>May</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>June</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>July</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>August</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>September</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>October</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>November</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>December</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QComboBox" name="cmbStartYear"/>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Duties</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Job Title</string>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QComboBox" name="cmbYearTo"/>
         </item>
         <item row="3" column="1" colspan="3">
          <widget class="QComboBox" name="cmbStartMonth">
           <item>
            <property name="text">
             <string>January</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>February</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>March</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>April</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>May</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>June</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>July</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>August</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>September</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>October</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>November</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>December</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QPushButton" name="btnAddDuty">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" rowspan="2">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Date Ended</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1" colspan="4">
          <widget class="QListView" name="listView"/>
         </item>
         <item row="8" column="1" colspan="4">
          <widget class="QTextEdit" name="textEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Date Started</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Company</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="4">
          <widget class="QLineEdit" name="txtLocation"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Location</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="4">
          <widget class="QLineEdit" name="txtCompanyName"/>
         </item>
         <item row="0" column="1" colspan="4">
          <widget class="QLineEdit" name="txtJobTitle"/>
         </item>
         <item row="9" column="2" colspan="3">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="7" column="4">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>Still In Role</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabEdu">
       <attribute name="title">
        <string>Education</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabVolunteer">
       <attribute name="title">
        <string>Volunteer Work</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabReferees">
       <attribute name="title">
        <string>Referees</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Base Cover Letter</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabApplications">
       <attribute name="title">
        <string>Job Applications</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionPrint_Applicant_Details"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionPrint_Applicant_Details">
   <property name="text">
    <string>Print Applicant Details</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>txtGivenName</tabstop>
  <tabstop>txtMiddleName</tabstop>
  <tabstop>txtSurname</tabstop>
  <tabstop>txtPreferredName</tabstop>
  <tabstop>txtAddress1</tabstop>
  <tabstop>txtAddress2</tabstop>
  <tabstop>txtCity</tabstop>
  <tabstop>txtPostcode</tabstop>
  <tabstop>txtState</tabstop>
  <tabstop>txtCountry</tabstop>
  <tabstop>txtEmail</tabstop>
  <tabstop>txtPhone</tabstop>
  <tabstop>txtWebsite</tabstop>
  <tabstop>txtLinkedIn</tabstop>
  <tabstop>chkIsMononymous</tabstop>
  <tabstop>chkNoMidName</tabstop>
  <tabstop>chkEastAsian</tabstop>
  <tabstop>chkPreferWest</tabstop>
  <tabstop>textPersonalProfile</tabstop>
  <tabstop>txtSkill</tabstop>
  <tabstop>btnAddSkill</tabstop>
  <tabstop>txtAchievement</tabstop>
  <tabstop>btnAddAchievement</tabstop>
  <tabstop>skillsListView</tabstop>
  <tabstop>achievementsListView</tabstop>
  <tabstop>btnSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
